nimbus/src/dbcache.rs

```diff
@@ -43,14 +44,21 @@ impl DatabaseCache {
         // By passing in the active `writer` we read the state of enrollments
         // as written by the calling code, before it's committed to the db.
         let experiments = get_enrollments(&db, &writer)?;
-        // Build the new hashmap.
-        let mut eb = HashMap::with_capacity(experiments.len());
+
+        // Build the new hashmaps
```

Comment:
I think a comment here about `feature_id`s, and that this is pretty temporary; only one experiment per feature (not multiple features per experiment) is implemented here, rather the in the `EnrollmentEvolver`.

Reply:
Added.